internal/node/node_loader.js

```diff
@@ -459,8 +459,10 @@ module.constructor._resolveFilename = function(request, parent) {
 // source-map-support.
 if (TEMPLATED_install_source_map_support) {
   try {
-    require('source-map-support').install();
-  } catch (e) {
+    const sourcemap_support_package = path.resolve(process.cwd(),
+          '../build_bazel_rules_nodejs/third_party/github.com/source-map-support');
+    require(sourcemap_support_package).install();
+  } catch (_) {
     console.error(`WARNING: source-map-support module not installed.
```

Comment:
let's remove this warning, I don't think we show it to the right person. Guard it with `if (DEBUG)`